libcxx/test/libcxx/headers_in_modulemap.sh.py

```diff
@@ -0,0 +1,51 @@
+# RUN: %{python} %s %{include}
+
+import pathlib
+import sys
+
+headers = list(pathlib.Path(sys.argv[1]).rglob("*"))
```

Comment:
Shouldn't we be using `libcxx/utils/libcxx/header_information.py` here instead?

Reply:
No. We want all headers in the modulemap, not just the public ones.

Reply:
We already have some of that logic in `header_information.py`, we should just add what we need to it and then use it here. For example, we already have the logic for excluding `libcxx.imp` and `module.modulemap`.

We probably want a slightly different list of headers but we can make clear that it's used for modulemap generation.